docs/guides/basic-rbac.mdx

```diff
@@ -47,48 +47,53 @@ declare global {
 
 ## Set the admin role for your user
 
-Later, you will add a basic admin tool to change the user's role, but for now, let's manually add a role to your own user account. In the Clerk Dashboard, navigate to [**Users**](https://dashboard.clerk.com/last-active?path=users) and select your own user account. Scroll down to the **Metadata** section and next to the **Public** option, select **Edit**. Add the following JSON and select **Save**.
+Later in the, you will add a basic admin tool to change a user's role. For now, let's manually add a role to your own user account.
+
+1. In the Clerk Dashboard, navigate to [**Users**](https://dashboard.clerk.com/last-active?path=users) and select your own user account.
+1. Scroll down to the **User metadata** section and next to the **Public** option, select **Edit**.
+1. Add the following JSON and select **Save**.
 
 ```json
 {
   "role": "admin"
 }
 ```
 
-![The Users page in the Clerk Dashboard with the 'Edit public metadata' modal open. The modal has a text field with the JSON 'role': 'admin'.](/docs/images/guides/basic-rbac/edit-public-metadata.webp)
+## Create and protect an admin dashboard
 
-## Create an admin dashboard and protect it
+Now that your user has the `admin` role, let's build an admin dashboard manage user roles easily.
 
-Now that your user has the `admin` role, let's build an admin dashboard to help you easily change users' roles. In your `app/` directory, create an `admin/` folder. Within the `admin/` folder, create a `dashboard/` folder. In the `dashboard/` folder, create a file named `page.tsx`. Copy the following code and paste it into the file.
+1. In your `app/` directory, create an `admin/` folder.
+1. Inside the `admin/` folder, create a `dashboard/` folder
+1. In the `dashboard/` folder, create a file named `page.tsx`.
+1. Copy the following code and paste it into the file.
 
-You want the dashboard to only be available to users with the `admin` role. [As you configured earlier](#configure-the-session-token), a user's role can be found in the metadata stored in the session token. You can access the session token's claims using Clerk's `auth()` hook.
+You want the dashboard to be accessible only to users with the `admin` role. As configured earlier in the [Configure the session token](#configure-the-session-token) section, a user's role is stored in the metadata within the session token. You can access the session token's claims using Clerk's `auth()` hook.
 
 ```tsx {{ filename: 'app/admin/dashboard/page.tsx' }}
 import { auth } from '@clerk/nextjs/server'
 import { redirect } from 'next/navigation'
 
-export default function AdminDashboard() {
+export default async function AdminDashboard() {
```

Comment:
```suggestion
export default function AdminDashboard() {
```
There is nothing that requires an `await` so this shouldn't be swapped to an `async` function.